indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java

```diff
@@ -410,6 +414,8 @@ public void run()
               plumber.finishJob();
             }
           }
+
+          FileUtils.forceDelete(firehoseTempDir);
```

Comment:
Should this be moved to the outer finally block? If an exception occurs after firehoseTempDir is created (normalExit is false), it looks like firehoseTempDir won't be deleted here.